examples/configs/neurobooth_os_config.json

```diff
@@ -1,26 +1,28 @@
 {
-    "data_out": "C:\\neurobooth\\neurobooth_data\\\\",
-    "nas": "Z:\\data\\\\",
-    "video_tasks": "C:\\Users\\STM\\Dropbox (Partners HealthCare)\\Neurobooth Videos for tasks\\Videos_to_present",
+    "remote_data_dir": "C:/data/",
+    "video_tasks": "C:/Users/STM/Dropbox (Partners HealthCare)/Neurobooth Videos for tasks/Videos_to_present",
     "cam_inx_lowfeed": 0,
-    "default_log_path": "D:\\neurobooth\\neurobooth_logs",
+    "default_log_path": "C:/neurobooth/neurobooth_logs",
     "acquisition": {
         "name": "acq",
         "user": "ACQ",
         "pass": "",
-	"port": 1280
+	    "port": 1280,
+        "local_data_dir": "C:/neurobooth/neurobooth_data/"
```

Comment:
For ACQ, should be D:/ instead of C:/ (check old value of the ACQ environment variable)

Reply:
Same as above